doc/source/install/ubuntu.md

```diff
@@ -120,3 +120,86 @@ sudo apt -V -y install groonga-normalizer-mysql
 Build from source is for developers.
 
 See {doc}`/install/cmake` .
+
+## Migration from Groonga PPA (ppa:groonga/ppa) to Groonga APT Repository (packages.groonga.org)
+
+This section guides you through the migration from the deprecated Groonga PPA
+(Personal Package Archive, ppa:groonga/ppa) to our new Groonga APT Repository
+(packages.groonga.org).
+
+By switching to the Groonga APT Repository, you'll receive packages built with
+Apache Arrow enabled. Which unlocks extra features such as parallel offline
+index building. We strongly recommend you to migrate to enjoy these enhancements.
+
+If you are currently using the Groonga PPA, please follow the
+steps below to switch to the new package source.
+
+### Configure the Groonga APT Repository
+
+#### Register Groonga APT Repository
+
+To register the Groonga APT Repository, install the `groonga-apt-source` package
+as follows.
+
+```bash
+sudo apt update
+sudo apt install -y -V groonga-apt-source
+sudo apt update
+```
+
+#### Remove the Existing Package Source
+
+Remove the old Groonga PPA:
+
+```bash
+sudo add-apt-repository -y --remove ppa:groonga/ppa
+sudo apt update
```

Comment:
Do we need this? I think that `add-apt-repository` runs `apt update` automatically.

```suggestion
```